enterprise/signals.py

```diff
@@ -315,6 +332,15 @@ def delete_enterprise_catalog_data(sender, instance, **kwargs):     # pylint: di
             exc_info=exc
         )
 
+    customer = instance.enterprise_customer
+    for channel in INTEGRATED_CHANNELS:
+        if channel.objects.filter(enterprise_customer=customer, active=True).exists():
+            try:
+                mark_orphaned_content_metadata_audit.delay()
+                continue
```

Comment:
could we actually `break` here instead?  this audit job runs for metadata across all channels, right?

Reply:
yes it does